ProgrammaticUI/Controllers/HomeViewController.swift

```diff
@@ -2,12 +2,13 @@ import UIKit
 
 class HomeViewController: UIViewController {
 
-    let stackView = UIStackView()
-    private var kmsLabel: UILabel!
-    private var caloriesLabel: UILabel!
-    private var durationLabel: UILabel!
-    let imageFitness = SAImageView(imageNamed: "fitness.png")
-    var insertData = InsertDataViewController()
+    private let stackView = UIStackView()
+    private let summaryLabel = SALabel(texto: "Summary", font: .boldSystemFont(ofSize: 40))
+    private var kmsLabel = SALabel(texto: "Kms: 0.0", font: .systemFont(ofSize: 20))
+    private var caloriesLabel = SALabel(texto: "Calories: 0", font: .systemFont(ofSize: 20))
+    private var durationLabel = SALabel(texto: "Duration: 00:00:00", font: .systemFont(ofSize: 20))
```

Comment:
There is a standardised way to group font sizes with tokens like this:

```swift
struct SALabelToken {
    static let heading: SALabelToken = .init(size: 20, weight: .regular, bold: false)
    static let headingHighlight: SALabelToken = .init(size: 20, weight: .regular, bold: true)
    static let title: SALabelToken = .init(size: 40, weight: .regular, bold: true)
    ....
    
    let size: CGFloat
    let weight: UIFont.weight
    let bold: Bool
    var font: UIFont = .systemFont(ofSize: 20)
```

This would be used to setup the label so that it handles the setup with said token:
```suggestion
    private var durationLabel = SALabel(text: "Duration: 00:00:00", token: .heading)
```

Please also note the usage of `text` instead of `texto`.